### PR TITLE
added initial design for selection interface

### PR DESCRIPTION
**Pull request by LStrouhal**
Initial setup of selector page, i.e. the page on which the user can select number of questions, category and level of difficulty for the current round.

### CODE REVIEW

File: frontend/src/components/CategorySelector.js [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+                    <MenuItem value={15}>History</MenuItem>
+                    <MenuItem value={16}>Politics</MenuItem>
+                    <MenuItem value={17}>Art</MenuItem>
+                    <MenuItem value={18}>Celebrities</MenuItem>
+                    <MenuItem value={19}>Animals</MenuItem>
+                    <MenuItem value={20}>Vehicles</MenuItem>
+                    <MenuItem value={21}>Entertainment: Comics</MenuItem>
+                    <MenuItem value={22}>Science: Gadgets</MenuItem>
+                    <MenuItem value={23}>Entertainment: Japanese Anime & Manga</MenuItem>
+                    <MenuItem value={24}>Entertainment: Cartoon & Animations</MenuItem>
```
Comment (fabianschmauder) [inline frontend/src/components/CategorySelector.js:39]: i would create an Array of [{value:1, label:"General Knowledge"} , {value:2, label:"Entertainment: Books"},.... and then add a mapping items -> <MenuItem

File: frontend/src/components/CategorySelector.js [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+    const style = useStyles();
+    const [category, setCategory] = useState("");
+
+    const handleChange = selection => setCategory(selection.target.value);
+
+    return (
+        <section>
+            <FormControl className={style.formControl}>
+                <Select className={style.Select} disableUnderline={true} onChange={handleChange}>
```
Comment (fabianschmauder) [inline frontend/src/components/CategorySelector.js:15]: i think there should be a possibility to set the selected default value. like value={category}

File: frontend/src/components/DifficultySelector.js [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+export default function DifficultySelector() {
+    const style = useStyles();
+    const [difficulty, setDifficulty] = useState("");
+
+    const handleChange = selection => setDifficulty(selection.target.value);
+
+    return (
+        <Wrapper>
+            <FormControl className={style.formControl}>
+                <Select className={style.Select} disableUnderline={true} onChange={handleChange}>
```
Comment (fabianschmauder) [inline frontend/src/components/DifficultySelector.js:15]: i would also set the default value to have a controlled component

File: frontend/src/components/NumberSelector.js [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+            <FormControl className={style.formControl}>
+                <Select className={style.Select} disableUnderline={true} onChange={handleChange}>
+                    <MenuItem className = {style.MenuItem} value={"10"}>10</MenuItem>
+                    <MenuItem className = {style.MenuItem} value={"15"}>15</MenuItem>
+                    <MenuItem className = {style.MenuItem} value={"20"}>20</MenuItem>
+                    <MenuItem className = {style.MenuItem} value={"25"}>25</MenuItem>
+                    <MenuItem className = {style.MenuItem} value={"30"}>30</MenuItem>
+                    <MenuItem className = {style.MenuItem} value={"35"}>35</MenuItem>
+                    <MenuItem className = {style.MenuItem} value={"40"}>40</MenuItem>
+                </Select>
```
Comment (fabianschmauder) [inline frontend/src/components/NumberSelector.js:22]: the same as for the other MenuItems ;)

**Review by fabianschmauder — COMMENTED** [review]
LGTM ;)

File: frontend/src/components/CategorySelector.js (hunk 1 above)
Comment (LStrouhal) [reply frontend/src/components/CategorySelector.js:39]: done

File: frontend/src/components/NumberSelector.js (hunk 4 above)
Comment (LStrouhal) [reply frontend/src/components/NumberSelector.js:22]: done, thank you

**Pull request merged**